build.gradle

```diff
@@ -154,3 +154,9 @@ signing {
         sign publishing.publications.mavenJava
     }
 }
+
+javadoc {
+    if(JavaVersion.current().isJava9Compatible()) {
```

Comment:
About this option isJava9Compatible means that compatibility with 9 is so important for javadoc?

Reply:
I just want to understand about this option.

Reply:
I'm trying to get the information about your question.
I'm sorry it takes time to get it.

Reply:
Thank you let me know if you get information.

Reply:
It seems from Java9, `HTML5` is available.
This change lets us easy to use a search feature.
https://blog.idrsolutions.com/2017/05/how-html5-javadocs-in-java-9-will-make-your-life-easier/
> In Java 8 and earlier, Javadocs are generated using HTML 4.01. While it’ll continue to be the default, with Java 9 comes the option to have your Javadocs generated in the much more modern HTML5.

Reply:
Thank you for your information.
I understood well, and could learn history about it, I want to say thank you again.
I'll approve this PR soon.

Reply:
But, should I add some conditions in this branch. e.g. `isJava10Compatible()`?
https://docs.gradle.org/current/javadoc/org/gradle/api/JavaVersion.html